en/drafts/originals/corpus-analysis-with-spacy.md

```diff
@@ -25,68 +25,71 @@ doi: XX.XXXXX/phen0000
 
-This lesson will explore a subset of documents from MICUSP: 67 Biology papers and 98 English papers. Writing samples in this select corpus belong to all seven MICUSP genres: Argumentative Essay, Creative Writing, Critique/Evaluation, Proposal, Report, Research Paper, and Response Paper. This select corpus and the associated metadata csv are available to download as part of the [lesson materials](https://github.com/mkane968/Corpus-Analysis-with-SpaCy/tree/main/lesson-materials). It has been culled from the larger corpus in order to investigate the differences between two distinct disciplines of academic writing (Biology and English). It is also a manageable size for the purposes of this lesson. 
+This lesson will explore a subset of documents from MICUSP: 67 Biology papers and 98 English papers. Writing samples in this select corpus belong to all seven MICUSP genres: Argumentative Essay, Creative Writing, Critique/Evaluation, Proposal, Report, Research Paper, and Response Paper. This select corpus [`.txt_files.zip`](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-analysis-with-spacy/txt_files.zip) and the associated [`metadata.csv`](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/corpus-analysis-with-spacy/metadata.csv) are available to download as sample materials for this lesson. The dataset has been culled from the larger corpus in order to investigate the differences between two distinct disciplines of academic writing (Biology and English). It is also a manageable size for the purposes of this lesson. 
 
-**Quick note on corpus size and processing speed:** spaCy is able to process jobs of up to 1 million characters, so it can be used to process the full MICUSP corpus, or other corpora containing hundreds or thousands of texts.  You are more than welcome to retrieve the entire MICUSP corpus with [this webscraping code](https://github.com/mkane968/webscraping_micusp/blob/main/MICUSP_Webscraping_11-22.ipynb) and using that dataset for the analysis. However, if your computer has more limited processing power or memory, or if you choose to use the Google Colab version of this lessson, you may encounter lags in runtime or run out of memory when your code is trying to process larger corpora. 
+**Quick note on corpus size and processing speed:** spaCy is able to process jobs of up to 1 million characters, so it can be used to process the full MICUSP corpus, or other corpora containing hundreds or thousands of texts.  You are more than welcome to retrieve the entire MICUSP corpus with [this webscraping code](https://github.com/mkane968/webscraping_micusp/blob/main/MICUSP_Webscraping_11-22.ipynb) and using that dataset for the analysis. However, if your computer has more limited processing power or memory, or if you choose to use the Google Colab version of this lesson, you may encounter lags in runtime or run out of memory when your code is trying to process larger corpora. 
```

Comment:
@mkane968 and @jrladd, Could you confirm that we should replace the [link to the `.ipynb` in Megan's personal repo](https://github.com/mkane968/webscraping_micusp/blob/main/MICUSP_Webscraping_11-22.ipynb) (line 62) with the link to [the notebook that is now saved in our /assets directory?](https://github.com/programminghistorian/ph-submissions/blob/2-copyedit-corpus-analysis-with-spacy/assets/corpus-analysis-with-spacy/corpus-analysis-with-spacy.ipynb)?

Reply:
As far as I can tell (and @mkane968 will know for sure), these notebooks aren't the same and so the link shouldn't be replaced. The one in our /assets directory is the full notebook for the corpus analysis code. The other is for webscraping the corpus specifically, and it contains different code and instructions.

Reply:
Yes, to confirm what @jrladd said, the notebook in line 62 is different than the full notebook in the assets folder. It is a code to web-scrape additional/different datasets from All of the other changes look great, thanks!

Reply:
Okay. What I notice is that this is the only `.ipynb` file linked in the lesson. Where are you directing readers to the [Colab notebook you created for our repo that replicates the lesson's code in active cells](https://github.com/programminghistorian/ph-submissions/blob/2-copyedit-corpus-analysis-with-spacy/assets/corpus-analysis-with-spacy/corpus-analysis-with-spacy.ipynb)?